pkg/distributor/field_detection.go

```diff
@@ -145,6 +153,30 @@ func labelsContainAny(labels labels.Labels, names []string) (string, bool) {
 	return "", false
 }
 
+// normalizeLogLevel normalizes log level strings to lowercase standard values
+func normalizeLogLevel(level string) string {
+	levelBytes := []byte(level)
```

Comment:
nit: Can we use `unsafeGetBytes(level)` here, since `levelBytes` is read-only and not mutated?